program-error/src/lib.rs

```diff
@@ -21,12 +22,12 @@ use {
         MISSING_REQUIRED_SIGNATURES, NOT_ENOUGH_ACCOUNT_KEYS, UNINITIALIZED_ACCOUNT,
         UNSUPPORTED_SYSVAR,
     },
-    solana_msg::msg,
-    solana_pubkey::PubkeyError,
-    std::convert::TryFrom,
+    solana_pubkey_error::PubkeyError,
```

Comment:
Same here, If we make `solana-pubkey-error` optional, we should add a feature here.